### PR TITLE
feat: Added ability to GiveNamedItem using the new CsItem Enum

### PR DESCRIPTION
**Pull request by LordFetznschaedl**
I have created an enum of all items I have found online with the item string. 
In addition the possibility was added for players to GiveNamedItem using the Enum.

Example for M4A4:
player.GiveNamedItem(CsItem.M4A1);

### CODE REVIEW

File: managed/CounterStrikeSharp.API/Modules/Utils/EnumUtils.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CounterStrikeSharp.API.Modules.Utils
+{
+    public static class EnumUtils
+    {
+        public static string GetEnumMemberAttributeValue<T>(T enumValue)
```
Comment (KillStr3aK) [inline managed/CounterStrikeSharp.API/Modules/Utils/EnumUtils.cs:12]: should be nullable if it can return null?

**Review by KillStr3aK — COMMENTED** [review]
I dont think this is a needed change in general, would be handy for ppl but maybe make the enum available in a library

**Review by roflmuffin — APPROVED** [review]

File: managed/CounterStrikeSharp.API/Modules/Utils/EnumUtils.cs (hunk 1 above)
Comment (LordFetznschaedl) [reply managed/CounterStrikeSharp.API/Modules/Utils/EnumUtils.cs]: String is a referenced type and thus can be already null. Good practice to have checks anyways before using them further. But yes, I have changed it to nullable string.

File: managed/CounterStrikeSharp.API/Modules/Utils/CsItem.cs [hunk 2]
```diff
@@ -0,0 +1,244 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CounterStrikeSharp.API.Modules.Utils
+{
+    public enum CsItem
```
[the remark this answers is not in the document]
Comment (LordFetznschaedl) [reply managed/CounterStrikeSharp.API/Modules/Utils/CsItem.cs:10]: The list comes from the internet. Just have forgotten to remove the defuser. this is done with the new commit

**Comment by roflmuffin** [comment]
The enum looks good, but could we move and namespace this under `Modules/Entities/Constants`, similar to ItemDefinitionIndex?

**Comment by LordFetznschaedl** [comment]
> The enum looks good, but could we move and namespace this under `Modules/Entities/Constants`, similar to ItemDefinitionIndex?

Change has been made.

**Comment by roflmuffin** [comment]
Legendary, thank you sir

**Pull request merged**